web_src/css/repository.css

```diff
@@ -591,87 +591,79 @@
-  display: flex;
-  align-items: center;
+.repository.view.issue .issue-title-header {
+  width: 100%;
+  padding-bottom: 4px;
 }
 
-.repository.view.issue .title .issue-title.edit-active h1 {
+.repository.view.issue .issue-title-buttons,
+.repository.view.issue .edit-buttons {
   display: flex;
-  width: 100%;
 }
 
 @media (max-width: 767px) {
-  .repository.view.issue .title .issue-title.edit-active {
+  .repository.view.issue .issue-title {
     flex-direction: column;
   }
-  .repository.view.issue .title .issue-title.edit-active h1 {
-    margin-right: 0;
-    margin-bottom: 1rem;
-    padding-right: 0;
-  }
-  .repository.view.issue .title .issue-title.edit-active h1 .ui.input input {
-    width: calc(100% - 2rem);
-  }
-  .repository.view.issue .title .issue-title.edit-active .edit-buttons {
-    padding-bottom: 1rem;
-    width: 100%;
-  }
-  .repository.view.issue .title .issue-title.edit-active .edit-buttons .button {
-    width: 100%;
-    margin-right: 0.5rem;
-  }
-  .repository.view.issue .title .issue-title.edit-active .edit-buttons .button:last-child {
-    margin-right: 0;
+  .repository.view.issue .edit-buttons {
+    margin-top: .5rem;
   }
 }
 
-.repository.view.issue .title .issue-title h1 {
+.repository.view.issue .issue-title {
+  display: flex;
+  align-items: center;
+  flex: 1;
+  margin-bottom: 8px;
+}
+
+.repository.view.issue .issue-title h1 {
+  display: flex;
+  align-items: center;
+  flex: 1;
+  width: 100%;
   font-weight: 300;
-  font-size: 2.3rem;
+  font-size: 32px;
+  line-height: 40px;
   margin: 0;
-  padding-right: 0.5rem;
+  padding-right: 0.25rem;
+  min-height: 41px; /* avoid layout shift on edit */
 }
 
-.repository.view.issue .title .issue-title h1 .ui.input {
+.repository.view.issue .issue-title h1 .ui.input {
   font-size: 0.5em;
-  width: 100%;
 }
 
-.repository.view.issue .title .issue-title h1 .ui.input input {
+.repository.view.issue .issue-title h1 .ui.input input {
   font-size: 1.5em;
-  padding: 6px 1rem;
-}
-
-.repository.view.issue .title .issue-title .edit-button {
-  float: right;
-  padding-left: 1rem;
+  padding: 2px .5rem;
 }
 
-.repository.view.issue .title .issue-title .edit-buttons {
-  display: flex;
-}
-
-.repository.view.issue .title .issue-title .index {
+.repository.view.issue .issue-title .index {
   color: var(--color-text-light-2);
 }
 
-.repository.view.issue .title .issue-title .label {
+.repository.view.issue .issue-title .label {
   margin-right: 10px;
 }
 
-.repository.view.issue .title .issue-title .edit-zone {
+.repository.view.issue .issue-title .edit-zone {
   margin-top: 10px;
 }
 
+.issue-state-label {
+  display: inline-block !important;
+  font-size: 14px !important;
+  font-weight: 500 !important;
+  line-height: 20px !important;
+  padding: 5px 12px !important;
+  border-radius : 6px !important;
+  text-align: center !important;
+  white-space: nowrap !important;
+  margin-left: 0 !important;
+  margin-right: 4px !important;
+}
```

Comment:
I guess it doesn't need to be so "important"? Remove this "issue-state-label" , it still looks good, just need a little padding?

![image](https://user-images.githubusercontent.com/2114189/235823274-841c433f-8094-4a27-b43d-e0c03a6e602e.png)

Reply:
Fomantic `.ui.label` is two classes so will always win over 1. I do like the big label more, but maybe it can be toned down by 1-2 px.

Reply:
Does `ui xxx label gt-p-2` work? No idea is it necessary to add many new styles.

Reply:
I think I might just make a fomantic-less class instead.

Reply:
I do not see it's worthy. Reusing the `ui color label` is the clearest solution at the moment. Unless we define our "label" in our framework.

We should avoid introducing too many temporary/one-time UI components.

Reply:
State label could be useful elsewhere, but not sure where currently. It's either a unique class or a bunch of `!important`. I think unique class may be useful in the future for other places and it will allow removal of Fomantic dependencies.

Reply:
`unique class` is not a framework level design.

We already have plenty of `ui.label.xxx` classes, reusing them is the clearest solution. 

Even if we want to remove Fomantic UI, we can continue using these class names.

Reply:
If I have to override 10+ CSS properties from the original class, I think it makes no sense to keep using it, because at that point, it barely has is anything in common with it any more.

Reply:
What's the problem of the `original class`? I have tested on my side, remove the `issue-state-label` class, the UI looks good enough? If you like padding, you can use `gt-p-xx`?